ts/screens/wallet/TransactionDetailsScreen.tsx

```diff
@@ -185,6 +215,9 @@ class TransactionDetailsScreen extends React.Component<Props> {
       ? this.props.wallets[transaction.idWallet]
       : undefined;
 
+    const transactionPSP =
```

Comment:
The PSP displayed can be inconsistent.
I complete payment1 using the wallet1 and PSP1.
I complete payment2 using wallet1 but choosing PSP2.
The wallet1 PSP is now PSP2.
If we render the transaction1 the PSP2 will be rendered :(